packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx

```diff
@@ -137,6 +137,7 @@ export function NumberFieldBase(
         data-invalid={props.isInvalid || undefined}
         data-positive={props.isPositive || undefined}
         data-has-start-addon={!!props.startVisual || undefined}
+        data-has-end-addon
```

Comment:
this is always here due to the buttons right?

Reply:
Yes exactly.